Add load balancing method to transport server config (#1479)
nginxinc/kubernetes-ingress: docs-web/configuration/transportserver-resource.md

     - The health check configuration for the Upstream. See the `health_check <https://nginx.org/en/docs/stream/ngx_stream_upstream_hc_module.html#health_check>`_ directive. Note: this feature is supported only in NGINX Plus.
     - `healthcheck <#upstream-healthcheck>`_
     - No

>    * - ``loadBalancingMethod``

Author: Yes, good catch

     - The health check configuration for the Upstream. See the `health_check <https://nginx.org/en/docs/stream/ngx_stream_upstream_hc_module.html#health_check>`_ directive. Note: this feature is supported only in NGINX Plus.
     - `healthcheck <#upstream-healthcheck>`_
     - No
   * - ``loadBalancingMethod``

>      - The method used to load balance the upstream servers. By default, connections are distributed between the servers using a weighted round-robin balancing method. See the `upstream <http://nginx.org/en/docs/stream/ngx_stream_upstream_module.html#upstream>`_ section for available methods and their details.

Reviewer: ```suggestion
     - The method used to load balance the upstream servers. See the `upstream <https://nginx.org/en/docs/stream/ngx_stream_upstream_module.html#upstream>`_ section for available methods and their details. To use the round-robin method, specify ``round_robin``. The ``hash`` method only supports ``$remote_addr`` variable with optional text and the consistent parameter. For example, ``hash ${remote_addr}``,  ``hash ${remote_addr}AAA``, ``hash $remote_addr consistent``. The default is ``random two least_conn``. 
```
